types/relay-compiler/lib/language/RelayLanguagePluginInterface.d.ts

```diff
@@ -2,6 +2,7 @@ import { Root, Fragment, GeneratedDefinition } from '../core/GraphQLIR';
 import { IRTransform } from '../core/GraphQLCompilerContext';
 import { GeneratedNode, RelayConcreteNode } from 'relay-runtime';
 import { ScalarTypeMapping } from './javascript/RelayFlowTypeTransformers';
+import { Schema } from '../core/GraphQLIRVisitor';
```

Comment:
Can we move the Schema type to '../core/Schema', so it is under the same location as in the relay-compiler package?

https://github.com/facebook/relay/blob/eb2dea7d9d610c432462f900c24ebcbb56de6a97/packages/relay-compiler/core/Schema.js#L109